docs/features/editor.md

```diff
@@ -0,0 +1,164 @@
+# IDE / Editor
```

Comment:
See [here](https://shark.fish/ty-lsp-features/features/editor) for a rendered version of this file